packages/cli/src/test/unit/init/init_test.ts

```diff
@@ -240,7 +238,7 @@ suite('init', () => {
             foo: 'TEST',
           }));
       helpers.registerDependencies(yeomanEnv, [[
-                                     helpers.createDummyGenerator(),
+                                     <any>function() {}, 
```

Comment:
yeoman-test installs yeoman-generator 2.x which is now incompatible with its `createDummyGenerator()` function because of the absence of Generator.extend() static method.

Simplest thing here to do was drop a meaningless stub function into the test register dependencies call.